src/app/services/notification.service.ts

```diff
@@ -23,6 +23,7 @@ import { Subject } from 'rxjs';
 export class NotificationService {
   public notification$: Subject<string> = new Subject<string>();
   public error$: Subject<string> = new Subject<string>();
+  public importantNotification$: Subject<string> = new Subject<string>();
```

Comment:
Why don't we just use `error$` for this, and make these into modal dialogs?

As noted in #39, I'm already concerned that errors aren't as visible as they should be, and this would give us an opportunity to fix that as well.

Reply:
done